testplan/runners/pools/child.py

```diff
@@ -217,7 +203,7 @@ def worker_loop(self):
                                 time.time() - hb_resp.data,
                             )
                         )
-                    self._to_heartbeat = self._pool_cfg.worker_heartbeat
+                    next_heartbeat = now + self._pool_cfg.worker_heartbeat
```

Comment:
now = time.time() is gotten at the first line in while loop. if it takes some time to execute line 190-205, e.g. worker respond slowly due to high CPU consuming, then `now + self._pool_cfg.worker_heartbeat` could  be smaller than the current time.time(). After getting the result of heartbeat check, no need to check it again. So I think it could be better written:

while True:
    if if self._pool_cfg.worker_heartbeat and time.time() > next_heartbeat:
        ....
        next_heartbeat = time.time() + self._pool_cfg.worker_heartbeat